index.js

```diff
@@ -113,12 +114,15 @@ module.exports = class Localdrive {
     const entry = await this.entry(key)
     if (entry) await this.del(key)
 
+    // note: drive has reversed args compared with fs.symlink
     const { filename: pointer } = keyResolve(this.root, key)
-    const { filename: target } = keyResolve(this.root, linkname)
+    const { filename } = keyResolve(this.root, linkname)
 
     const release = await this._lock()
     try {
       await fsp.mkdir(path.dirname(pointer), { recursive: true })
+
+      const target = path.isAbsolute(linkname) ? filename : path.relative(this.root, filename)
```

Comment:
const target = linkname.startsWith('/') ? path.join(this.root, linkname) : linkname.replace(/\//g, path.sep)